u2-routing-controllers/app.js

```diff
@@ -0,0 +1,37 @@
+// import opject in file
+const express = require('express')
+const { urlencoded, json } = require('body-parser')
+const { getUserByQuery, getUserByPath, getUserByForm, search } = require('./features/userController')
+const { logger } = require('./_services/logger')
+const { index } = require('./features/indexController')
+const { getproduct } = require('./features/productController')
+const { getrgb } = require('./features/rgbController')
+const { gettemp } = require('./features/tempController')
+const { getecho, getechoByform } = require('./features/echoController')
```

Comment:
ฉันแนะนำให้คุณใช้ `lowerCamelCase` สำหรับชื่อตัวแปร:
I recommend that you use `lowerCamelCase` for variable names:
- `getProduct`
- `getRgb` or `getRGB` (I prefer the first)
- `getTemp`
- `getEcho`
- `getEchoByForm`